lib/pronto/config.rb

```diff
@@ -4,7 +4,7 @@ def initialize(config_hash = ConfigFile.new.to_h)
       @config_hash = config_hash
     end
 
-    %w(github gitlab).each do |service|
+    %w(github gitlab text_formatter).each do |service|
```

Comment:
Spot on!

I've replaced it with `configurable` because I just don't fine something like the following as nice:

```ruby
@config_hash[nested_config][key]
```